affected_purls:
- pkg:pypi/torch@1.0.0
- pkg:pypi/torch@1.0.1
- pkg:pypi/torch@1.0.1.post2
- pkg:pypi/torch@1.1.0
- pkg:pypi/torch@1.10.0
- pkg:pypi/torch@1.10.1
- pkg:pypi/torch@1.10.2
- pkg:pypi/torch@1.1.0.post2
- pkg:pypi/torch@1.11.0
- pkg:pypi/torch@1.12.0
- pkg:pypi/torch@1.12.1
- pkg:pypi/torch@1.13.0
- pkg:pypi/torch@1.2.0
- pkg:pypi/torch@1.3.0
- pkg:pypi/torch@1.3.0.post2
- pkg:pypi/torch@1.3.1
- pkg:pypi/torch@1.4.0
- pkg:pypi/torch@1.5.0
- pkg:pypi/torch@1.5.1
- pkg:pypi/torch@1.6.0
- pkg:pypi/torch@1.7.0
- pkg:pypi/torch@1.7.1
- pkg:pypi/torch@1.8.0
- pkg:pypi/torch@1.8.1
- pkg:pypi/torch@1.9.0
- pkg:pypi/torch@1.9.1
aliases:
- CVE-2022-45907
- GHSA-47fc-vmwq-366v
fixed_purl: []
references:
- id: 173686
  reference_id: ''
  url: https://github.com/pytorch/pytorch/commit/767f6aa49fe20a2766b9843d01e3b7f7793df6a3
- id: 173685
  reference_id: ''
  url: https://github.com/pytorch/pytorch/issues/88868
- id: 173687
  reference_id: ''
  url: https://github.com/pytorch/pytorch/issues/89855
- id: 173688
  reference_id: ''
  url: https://github.com/pytorch/pytorch/pull/89189
- id: 173689
  reference_id: ''
  url: https://github.com/pytorch/pytorch/releases/tag/v1.13.1
- id: 173684
  reference_id: CVE-2022-45907
  url: https://nvd.nist.gov/vuln/detail/CVE-2022-45907
- id: 173690
  reference_id: GHSA-47fc-vmwq-366v
  url: https://github.com/advisories/GHSA-47fc-vmwq-366v
severities:
- id: 27199
  reference_id: 173690
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: CRITICAL
summary: PyTorch vulnerable to arbitrary code execution
vulnerability_id: VCID-2n95-1v7w-aaab
weaknesses: []
